examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx

```diff
@@ -254,9 +260,9 @@ export const BreakoutManager: React.FC = () => {
               variant="solid-danger"
               size="x-large"
             >
-              Stop session
+              Stop session{" "}
```

Comment:
*SUPER NIT*
the space should be in the template string
```ts
                ? ` (${formatDisplayTime(timer.restDuration)})`
```

Reply:
Good catch!